xonsh/tokenize.py

```diff
@@ -83,9 +84,7 @@
 from xonsh.lazyasd import LazyObject
 from xonsh.platform import PYTHON_VERSION_INFO
 
-HAS_WALRUS = PYTHON_VERSION_INFO > (3, 8)
-if HAS_WALRUS:
-    from token import COLONEQUAL  # type:ignore
+HAS_WALRUS = True
```

Comment:
I think we can remove this entirely, and then skip the checks on lines 128 and 235.

Reply:
done